Sources/EdgeBridge.swift

```diff
@@ -103,7 +108,7 @@ public class EdgeBridge: NSObject, Extension {
         }
 
         let xdmEventData: [String: Any] = [
-            "data": mappedData,
+            "data": formatData(data),
```

Comment:
why are we calling formatData again here?

Reply:
I must have missed this during the branch rebase. There were so many merge conflicts. I've reverted this back to `mappedData`.